projects/vdk-plugins/vdk-ipython/README.md

```diff
@@ -14,19 +14,21 @@ Then to load the extension in Jupyter the user should use:
 ```
 %reload_ext vdk_ipython
 ```
-And to load the job_input:
+And to load the data_job (Job Control object):
```

Comment:
Having a "data job" from the perspective of:
a) sources the users create (by template, or from scratch)
b) at the same time, a notebook runtime-loaded data job context
, could be confusing. For example, a user will be referring to "data job" and it would not be clear what he or she is referring to exactly;

Reply:
I changed it to VDK, as mentioned in https://github.com/vmware/versatile-data-kit/pull/1678#discussion_r1120069027 
I forgot to update the README. I will update it now